File: src/codes/Users_Manual/modulation/csv_0.cpp
//~#include<autd3.hpp>
//~#include "autd3/modulation/audio_file/csv.hpp"
#include "autd3/modulation/audio_file.hpp"

//~int main() {
const auto path = "path/to/foo.csv";
const auto m = autd3::modulation::audio_file::Csv(path, 4000 * autd3::Hz);
//~return 0; }

File: src/codes/Users_Manual/modulation/csv_0.rs
# use autd3::prelude::*;
use autd3_modulation_audio_file::Csv;

# #[allow(unused_variables)]
# fn main() -> Result<(), Box<dyn std::error::Error>> {
let path = "path/to/foo.csv";
let m = Csv::new(&path, 4000 * Hz);
# Ok(())
# }